### PR TITLE
[FR] rust french translation chap7

### PR DESCRIPTION
**Pull request by mgemard**

### CODE REVIEW

File: lessons.yaml [hunk 1]
```diff
@@ -4986,6 +4986,16 @@ pages:
...
 
+    fr:
+      title: Chapitre 7 - Programmation Orientée Object
+      content_markdown: |
+        L'utilisation de fonctions pour exprimer une idée ou modéliser le comportement
+        et les données d'un programme a toujours été une manière éprouvée d'écrire un
+        programme (C le fait depuis des décennies!). Cependant, il existe d'autre paradigme
+        de programmation permetant l'agrégation et l'abstraction de données. Vous connaissez
+        peut-être déjà la programmation orientée objet (POO). Dans ce chapitre, nous
+        explorerons le langage de programmation Rust au-delà des fonctions.
```
Comment (VincentFoulon80) [inline lessons.yaml:4997]:
```suggestion
        L'utilisation de fonctions pour exprimer une idée ou modéliser le comportement
        et les données d'un programme a toujours été une manière éprouvée d'écrire un
        programme (C le fait depuis des décennies!). Cependant, il existe d'autres paradigmes
        de programmation permettant l'agrégation et l'abstraction de données. Vous connaissez
        peut-être déjà la programmation orientée objet (POO). Dans ce chapitre, nous
        explorerons le langage de programmation Rust au-delà des fonctions.
```

File: lessons.yaml [hunk 2]
```diff
@@ -5020,6 +5044,17 @@ pages:
 
         That said, Rust implements many programming language features that you might not mind this lacking.
 
+    fr:
+      title: Rust n'est pas POO
+      content_markdown: |
+        Il n'y a pas d'héritage avec Rust.
+
+        * Les structures ne peuvent pas hériter des champs d'une structure parent.
```
Comment (VincentFoulon80) [inline lessons.yaml:5052]:
```suggestion
        * Les structures ne peuvent pas hériter des champs d'une structure parente.
```

File: lessons.yaml [hunk 3]
```diff
@@ -5020,6 +5044,17 @@ pages:
...
+        * Les structures ne peuvent pas hériter des fonctions d'une structure parent.
```
Comment (VincentFoulon80) [inline lessons.yaml:5053]:
```suggestion
        * Les structures ne peuvent pas hériter des fonctions d'une structure parente.
```

File: lessons.yaml [hunk 4]
```diff
@@ -5002,6 +5012,20 @@ pages:
         * Polymorphism - The ability to interacting with an object from different functional perspectives.
         * Inheritence - The ability to inherit data and behavior from other objects.
 
+    fr:
+      title: Qu'est-ce que la POO?
+      content_markdown: |
+        La programmation orientée objet se réfère grossièrement aux langages de
+        programmation qui possèdent un certain nombre des fonctionnalités suivantes:
+        * Encapsulation - Association de champs et de fonctions dans une seule unité
+        appelée *objet*.
```
Comment (VincentFoulon80) [inline lessons.yaml:5021]:
```suggestion
        * Encapsulation - Association de champs et de fonctions dans une seule unitée
        appelée *objet*.
```

**Comment by VincentFoulon80** [comment]
Oh, this was already merged.

**Comment by richardanaya** [comment]
Hmm, no sure how to  get those changes in now

**Comment by VincentFoulon80** [comment]
I'll create a pull request 😃 i'm currently editing the file

**Pull request merged**